src/main/webapp/app/entities/programming-exercise/programming-exercise.route.ts

```diff
@@ -120,6 +133,16 @@ export const programmingExercisePopupRoute: Routes = [
         canActivate: [UserRouteAccessService],
         outlet: 'popup',
     },
+    {
+        path: 'course/:courseId/programming-exercise-import',
```

Comment:
please open a popup without a route. Look into the latest delete dialog changes

Reply:
Popup now doesn't have a route anymore